sympy/holonomic/holonomic.py

```diff
@@ -619,8 +620,39 @@ def integrate(self, limits, initcond=False):
         # to get the annihilator, just multiply by Dx from right
         D = self.annihilator.parent.derivative_operator
 
+        # if the function have initial conditions of the series format
+        if self.singular_ics:
+
```

Comment:
PEP 8 says:

     Use blank lines in functions, sparingly, to indicate logical sections.

There is quite a number in this function :)